lib/test/cdo/test_buffer.rb

```diff
@@ -81,16 +82,14 @@ def test_fork
   end
 
   def test_min_interval
-    b = TestBuffer.new(batch_count: 1, min_interval: 0.1)
-    start = Concurrent.monotonic_time
-    4.times {b.buffer('foo')}
-    b.flush!
-    finish = Concurrent.monotonic_time
-    assert_equal 4, b.flushes
-    time_taken = finish - start
-    # Given a min_interval of 1 second and 4 flushes, assert the flush! takes 3-4 seconds.
-    assert_operator time_taken, :>, 0.3
-    assert_operator time_taken, :<, 0.4
+    n = 4
+    interval = 0.1
+
+    b = TestBuffer.new(batch_count: 1, min_interval: interval)
+    Array.new(n) {Thread.new {b.buffer('foo')}}.each(&:join)
+    # Assert the flush! takes the expected amount of time to complete.
+    assert_in_delta (n - 1) * interval, Benchmark.realtime {b.flush!}, interval / 2
```

Comment:
I think this might be wrong. This ends up testing `.25 <= time_taken <= .35` but it should be `.3 <= time_taken <= .4`

Reply:
Oh ok, if we want to ensure that the amount of time is strictly no less than what we expect, the explicit operators are more expressive than a rough delta/epsilon here. I'll revert this part of the test change.